src/main/resources/i18n/Enums_en.properties

```diff
@@ -19,6 +19,18 @@ accelerator.DeliverableCategory.SupplementalFiles=Supplemental Files
 accelerator.DeliverableCategory.VerraNonPermanenceRiskToolNPRT=Verra Non-Permanence Risk Tool (NPRT)
 # Noun, as in a legal document
 accelerator.DeliverableType.Document=Document
+accelerator.ScoreCategory.Carbon=Carbon
+accelerator.ScoreCategory.Finance=Finance
+accelerator.ScoreCategory.Forestry=Forestry
+accelerator.ScoreCategory.Legal=Legal
+accelerator.ScoreCategory.Community=Community
+accelerator.ScoreCategory.GIS=GIS
+accelerator.ScoreCategory.ClimateImpact=Climate Impact
+accelerator.ScoreCategory.ExpansionPotential=Expansion Potential
+accelerator.ScoreCategory.ExperienceAndUnderstanding=Experience and Understanding
+accelerator.ScoreCategory.OperationalCapacity=Operational Capacity
+accelerator.ScoreCategory.ResponsivenessAndAttentionToDetail=Responsiveness and Attention to Detail
+accelerator.ScoreCategory.ValuesAlignment=Values Alignment
```

Comment:
These are all only for internal use, so I think we can include `isLocalizable = false` as an argument to the enum entry in `Config.kt` and then we don't need these strings in the strings table.